internal/config/config.go

```diff
@@ -31,20 +32,24 @@ type NeutronQueryRelayerConfig struct {
 
 const EnvPrefix string = "RELAYER"
 
+// NeutronChainConfig TODO: research if HomeDir parameter is needed for something but keys (so it might be optional and ignored for memory keyring)
```

Comment:
it's passed to the `GetNeutronChain` besides its keyring's usage. you can also inspect your `~/.neutrond` directory which is the default value for `neutrond` cli calls and probably come to a conclusion about this TODO:
```
neutrond --help

...
      --home string         directory for config and data (default "~/.neutrond")
...
```

Reply:
Yeah, I mean of course it's passed there but _is it actually needed_ there?